urnai/sc2/environments/sc2environment.py

```diff
@@ -61,26 +48,33 @@ def __init__(
         self.realtime = realtime
         self.done = False
 
-        self.start()
+        self.start(action_space, use_raw_units, raw_resolution, 
+                   use_feature_units, screen, minimap)
 
-    def start(self):
+    def start(
+            self,
+            action_space=actions.ActionSpace.RAW,
+            use_raw_units=True,
+            raw_resolution=64,
+            use_feature_units=True,
+            screen=64,
+            minimap=64,
```

Comment:
The default parameter values are redundant. I recommend either setting everything to `None` and checking during its assignment in `sc2_env.SC2Env` or change `start` to `_start`.

Reply:
> The default parameter values are redundant. I recommend either setting everything to `None` and checking during its assignment in `sc2_env.SC2Env` or change `start` to `_start`.

In the first suggestion, would that require an `if-else` statement in every parameter of `sc2_env.SC2Env`? 
And in the second, changing `start` to `_start` would make the method private, only being callable from the `__init__` function, right? Is it necessary for the user to be able to call `start` directly?

Reply:
I think I've got it wrong. `start` will be called both in `__init__` and by the developer, correct? If so, you can leave it as it is.